cluster-autoscaler/go.mod

```diff
@@ -390,36 +390,36 @@ replace (
 	gotest.tools => gotest.tools v2.2.0+incompatible
 	gotest.tools/v3 => gotest.tools/v3 v3.0.2
 	honnef.co/go/tools => honnef.co/go/tools v0.0.1-2020.1.3
-	k8s.io/api => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/api
-	k8s.io/apiextensions-apiserver => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/apiextensions-apiserver
-	k8s.io/apimachinery => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/apimachinery
-	k8s.io/apiserver => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/apiserver
-	k8s.io/cli-runtime => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/cli-runtime
-	k8s.io/client-go => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/client-go
-	k8s.io/cloud-provider => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/cloud-provider
-	k8s.io/cluster-bootstrap => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/cluster-bootstrap
-	k8s.io/code-generator => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/code-generator
-	k8s.io/component-base => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/component-base
-	k8s.io/component-helpers => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/component-helpers
-	k8s.io/controller-manager => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/controller-manager
-	k8s.io/cri-api => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/cri-api
-	k8s.io/csi-translation-lib => /tmp/ca-update-vendor.NGE2/kubernetes/staging/src/k8s.io/csi-translation-lib
+	k8s.io/api => k8s.io/api v0.20.6
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.6
+	k8s.io/apimachinery => k8s.io/apimachinery v0.20.6
+	k8s.io/apiserver => k8s.io/apiserver v0.20.6
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.20.6
+	k8s.io/client-go => k8s.io/client-go v0.20.6
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.20.6
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.20.6
+	k8s.io/code-generator => k8s.io/code-generator v0.20.6
+	k8s.io/component-base => k8s.io/component-base v0.20.6
+	k8s.io/component-helpers => k8s.io/component-helpers v0.20.6
+	k8s.io/controller-manager => k8s.io/controller-manager v0.20.6
+	k8s.io/cri-api => k8s.io/cri-api v0.20.6
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.20.6
```

Comment:
What happens if you remove these lines now? You are replacing a version with the same version, so I think you can get rid of these. Same for the others below and the final K8s.io replacement

Reply:
should work, i'll give it a try

Reply:
i tried removing these from the `replace` section and when i run `go mod tidy` i get this error

```
go: k8s.io/kubernetes@v1.20.6 requires
        k8s.io/api@v0.0.0: reading k8s.io/api/go.mod at revision v0.0.0: unknown revision v0.0.0
```